Add Smart Control guidance to Install .NET on Windows / Troubleshooting (#39479)
dotnet/docs: docs/core/install/windows.md

    :::image type="content" source="media/windows/edit-vars.png" alt-text="The environment variables list for the system.":::

### Building apps is slower than expected

> Check if the Smart App Control Windows feature is off. Smart App Control is not recommended to be enabled on machines used for development. It should be off as any other setting may negatively impact SDK performance.

Reviewer: ```suggestion
Ensure that the Smart App Control Windows feature is off. Smart App Control isn't recommended to be enabled on machines used for development. It should be off as any other setting may negatively impact SDK performance.
```

> Ensure that the Smart App Control Windows feature is off. Smart App Control isn't recommended to be enabled on machines used for development. It should be off as any other setting may negatively impact SDK performance.

Reviewer 2: ```suggestion
Ensure that Smart App Control, a Windows feature, is off. Smart App Control isn't recommended to be enabled on machines used for development. Any setting other than "off" might negatively impact SDK performance.
```
